### ISSUE TITLE
Added a health check to the Dockerfile

### ISSUE THREAD

**Issue by pgerhard** [maildev/maildev#264]
Firstly thanks for creating this project and all your hard work. 

I am currently interested in using the docker image during software development and in order to do this efficiently, I need to have health information for the docker container.
Since there was already a health endpoint available I added a health check to the Dockerfile based on this endpoint.

**Comment by pgerhard** [comment]
@DominikSerafin: I don't believe that this ticket has been fixed. Docker image tagged as "latest" has yet to include the health endpoint I created over 2 years ago. And yes, this was working an didn't require a "fix" contrary to the changes made [here](https://github.com/maildev/maildev/commit/114fa5b84f5a49564b9a4a6e8b039a65c0cd94ca). However I can see the value of not installing an additional package if `wget` is already available.

The last update on this was that it will be included "soon in the production release" (https://github.com/maildev/maildev/pull/265#issuecomment-531495487)

When can we expect to finally get this feature in the image?
